.stylelintrc

```diff
@@ -147,6 +147,7 @@
       "ignoreAtRules": ["if", "else"]
     } ],
     "at-rule-no-unknown": null,
+    "selector-type-no-unknown": null,
```

Comment:
Do we really want to disable it for all?
They support ignoring `custom-elements`:
https://stylelint.io/user-guide/rules/selector-type-no-unknown/#ignore-custom-elements-default-namespace

Reply:
i think so, but open for discussions